collect_app/src/main/java/org/odk/collect/android/formentry/audit/AuditEventLogger.java

```diff
@@ -136,19 +140,19 @@ private boolean isDuplicatedIntervalEvent(AuditEvent newAuditEvent) {
      */
     public void exitView() {
         if (isAuditEnabled()) {
-            manageSavedEvents();
+            manageSavedEvents(formController);
             writeEvents();
         }
     }
 
     // Filter all events and set final parameters of interval events
-    private void manageSavedEvents() {
+    private void manageSavedEvents(FormController formController) {
```

Comment:
Is there really an advantage to also having `FormController` come in to the methods if it's also a field?

Reply:
Not sure. This is probably something I left around while reworking. Good catch.